csvcubed/csvcubed/readers/cubeconfig/v1/columnschema.py

```diff
@@ -78,20 +90,48 @@ def map_to_new_qb_dimension(
         )
         # The NewQbCodeList and Concepts are populated in the NewQbDimension.from_data() call
         # the _get_code_list method overrides the code_list if required.
-        new_dimension.code_list = self._get_code_list(new_dimension, csv_column_title)
+        new_dimension.code_list = self._get_code_list(
+            new_dimension, csv_column_title, cube_config_path=config_path
+        )
         return new_dimension
 
     def _get_code_list(
-        self, new_dimension: NewQbDimension, csv_column_title: str
+        self,
+        new_dimension: NewQbDimension,
+        csv_column_title: str,
+        cube_config_path: Optional[Path],
     ) -> Optional[QbCodeList]:
-
         if isinstance(self.code_list, str):
             if looks_like_uri(self.code_list):
                 return ExistingQbCodeList(self.code_list)
+            # The following elif is for cube config v1.1. This also requires the user to define the configuration in the build command, and therefore cube_config_path.
+            elif cube_config_path and code_list_config_json_exists(
+                Path(self.code_list), cube_config_path.parent
```

Comment:
@robons If you agree, I would like to do this as a new ticket. Let me know.

Reply:
I was hoping it was something we could try out pretty quickly tbh. We already know how to reference other schema definitions in separate files, I was hoping it would be very similar.

Reply:
Ok, that makes sense. Thanks for the info!